src/main/java/wooteco/subway/admin/dto/LineStationResponse.java

```diff
@@ -0,0 +1,54 @@
+package wooteco.subway.admin.dto;
+
+import wooteco.subway.admin.domain.LineStation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LineStationResponse {
+
+    private Long id;
+    private Long preStationId;
+    private Long stationId;
+    private int distance;
+    private int duration;
+
+    public LineStationResponse(Long id, Long preStationId, Long stationId, int distance, int duration) {
+        this.id = id;
+        this.preStationId = preStationId;
+        this.stationId = stationId;
+        this.distance = distance;
+        this.duration = duration;
+    }
+
+    public static LineStationResponse of(LineStation lineStation) {
+        return new LineStationResponse(lineStation.getId(), lineStation.getPreStationId(), lineStation.getStationId(),
+                lineStation.getDistance(), lineStation.getDuration());
+    }
+
+    public static List<LineStationResponse> listOf(List<LineStation> lineStations) {
+        List<LineStationResponse> lineStationResponses = new ArrayList<>();
+        lineStations.forEach(lineStation -> lineStationResponses.add(of(lineStation)));
+        return lineStationResponses;
```

Comment:
```suggestion
        return lineStations.stream()
                .map(LineStationResponse::of)
                .collect(Collectors.toList());
```

값의 변환의 경우 Stream의 최종 연산을 사용하는 것이 좋습니다.

Reply:
감사합니다. 참고하여 리팩토링 하겠습니다 :)